Makefile

```diff
@@ -167,7 +167,11 @@ endif
 
 .PHONY: run-int-tests
 run-int-tests: install-kuttl-plugin vdb-gen ## Run the integration tests
+ifeq (${PARALLEL}, )
```

Comment:
Can we name the env. var E2E_PARALLELISM?  Also, instead of `ifeq` here, can we just define E2E_PARALLELISM with the `?=` setting so it is set unless overridden.

```
E2E_PARALLELISM?=2
```

We can include this near the top of the Makefile with the other defines and include a comment to document what behaviour is changes.

Reply:
Ok. Do you like how we are setting the env var through workflow_dispatch event?

Reply:
I think its okay.  I'm just not sure how often we would use it.  As far as I understood, the machines we use for GitHub CI are limited in size.  So increasing the parallelism made it more likely that we would see test failures due to timeouts.